data/data.go

```diff
@@ -200,3 +211,10 @@ func DBPath(id string) string {
 	relativeDataPath := config.GetString(configDataPathKey)
 	return path.Join(storagePath, relativeDataPath, id, "sqlite3")
 }
+
+func printDBStatsInfo(db *sql.DB, versionedId string) {
+	tick := time.Duration(statCollectionInterval * time.Second)
+	for _ = range time.Tick(tick) {
```

Comment:
Could you add a `select` and a quit channel? Func `Delete` will be the sender of the quit channel. If a db version is deleted by `Delete`, this will throw null pointer panic.

Reply:
Oh. Missed seeing your review comment. Am fixing it, good catch. Btw, I don't see ReleaseDB() being used at all. That is a bug, btw. I will open a ticket for it.

Reply:
I think I'm using `Delete` or `ReleaseDB` in some test code.

Reply:
It is not happening. I just checked. Can you open a ticket, and fix it? It should be straightforward.